tests/tests_suite_test.go

```diff
@@ -318,6 +320,9 @@ func createApp(name string, options ...string) *Session {
 	Expect(err).NotTo(HaveOccurred())
 	Eventually(cmd).Should(Say("created %s", name))
 	Eventually(cmd).Should(Exit(0))
+	Eventually(cmd).Should(SatisfyAll(
```

Comment:
Perhaps `"remote available at "` can stay since *I think* this will always show, but `"Git remote deis added"` needs to move to specific test(s) invoking `createApp(...)`